server/chatbotDashboard.mst

```diff
@@ -153,7 +153,7 @@
 
 <body>
     <nav class="navbar navbar-expand-md">
-        <a class="navbar-brand" id="navbar-brand">Brave Chatbot Dashboard</a>
+        <a class="navbar-brand" id="navbar-brand" href="/dashboard">Brave Chatbot Dashboard</a>
```

Comment:
This is probably unnecessary, and might be confusing if people accidentally click the page title.